docs/source/mb_specification.rst

```diff
@@ -13,11 +13,12 @@ This specification defines the directory structure a bundle must have and the ne
 Directory Structure
 ===================
 
-A MONAI Bundle is defined primarily as a directory with a set of specifically named subdirectories containing the model and metadata files. The root directory should be named for the model, given as "ModelName" in this example, and should contain the following structure:
+A MONAI Bundle is defined primarily as a directory with a set of specifically named subdirectories containing the model, metadata files and license. The root directory should be named for the model, given as "ModelName" in this example, and should contain the following structure:
 
 ::
 
   ModelName
+  ┣━ LICENSE
```

Comment:
The optional file `license.txt` is already mentioned in the docs directory, I would leave it there and make it mandatory if you like (and rename it `LICENSE`). If we want a separate license file for the data that should be `DATA_LICENSE` or something, having `LICENSE` in one place and `license.txt` in another seems confusing.

Reply:
Hi @ericspod, the `LICENSE` is for the entire bundle, and it appears to be a must-have file for a bundle. 
https://github.com/Project-MONAI/model-zoo/blob/bdd183ebc113a70d50db4e4fb2ebd9a7c8c7532b/ci/verify_bundle.py#L28
The `*license.txt` file, which appears to be an optional license, for a specific component of the bundle, such as data.
Does it makes sense?

Reply:
Yes I see what's been done so the verification hasn't matched the bundle's specification for a while then. The specification had said `license.txt` was for the model, we definitely should have separate licenses for model and data. We've let it get out of sync. I would say then we should change the verification as well but that's going to require modifying existing stored bundles. I still feel having all files in subdirectories is better organisation.

Reply:
If there is no particular reason to place the license in the root directory, I can assist in later modifying the existing bundle.
Could you please help confirm here? cc @yiheng-wang-nv @wyli @Nic-Ma

Reply:
Maybe we were thinking about the GitHub default license location for the bundles https://docs.github.com/en/repositories/managing-your-repositorys-settings-and-features/customizing-your-repository/licensing-a-repository#determining-the-location-of-your-license

If we release individual bundle as GitHub repo, the current location is consistent with that default..

Reply:
It's weird that Github will find the `README.md` file if it's in a subdirectory but not `LICENSE`, see this test: https://github.com/ericspod/BundleTestRepo

Reply:
Found some explanations about it https://github.com/licensee/licensee/issues/335 makes sense to me

Reply:
I understand the rationale somewhat but I still don't see justification for forcing people to do things their way. Make it an option like others have requested, look for the file in the root first then in docs. We can't change it though so we'll have to live with it, I don't know if symlinking to the file in docs would work so if it's critical Github display the license we have to leave it in the root.